compound-comptroller-bot/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "neth-comp",
```

```suggestion
  "name": "neth-comp-pause-event",
```
Re-reviewing it, perhaps it can be a bit more specific. I should've considered this previously.